scripts/compose.py

```diff
@@ -1841,7 +1886,7 @@ def _content(self):
                 "DATABASE_DIALECT=POSTGRESQL",
                 "DATABASE_DRIVER=org.postgresql.Driver",
                 "REDIS_URL=redis://redis:6379",
-                "ELASTICSEARCH_URL=http://elasticsearch:9200",
+                "ELASTICSEARCH_URL={}".format(self.es_urls),
```

Comment:
like the logstash case, needs to be joined or an entry picked

Reply:
https://github.com/elastic/apm-integration-testing/pull/557#discussion_r306676939